packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -380,8 +380,10 @@ export interface PluginConfigTypeIos {
 
   /**
    * Enables support for precompiled React Native iOS dependencies (`ReactNativeDependencies.xcframework`).
+   * When set to `true`, React Native will be built from source.
    * This feature is available from React Native 0.80 and later when using the new architecture.
-   * From React Native 0.81, this setting will also use a precompiled React Native Core (`React.xcframework`).
+   * Beginning with React Native 0.81, setting this to `false` will also enable the use
+   * of the precompiled React Native Core (`React.xcframework`).
```

Comment:
The default value is `false`, should we then omit the sentence "Beginning with React Native 0.81, setting this to `false` will also enable the use of the precompiled React Native Core (`React.xcframework`)."?

Reply:
That was what confused me in the first place

`Enables support for precompiled React Native iOS dependencies` suggests to me that setting this to `true` will use precompiled react native. 

But I agree that this sentence is redundant. 

What do you think about

```diff
-  * When set to `true`, React Native will be built from source.
+ * Setting this value to `true` will enable building from source and disable the use of precompiled React Native.
```